Hymns/ContentView.swift

```diff
@@ -1,33 +1,30 @@
-//
-//  ContentView.swift
-//  Hymns
-//
-//  Created by Luke Lu on 3/23/20.
-//  Copyright © 2020 skywalkerdude. All rights reserved.
-//
-
 import SwiftUI
 
 struct ContentView: View {
-    var hymns = hymnTestData
-    @State private var searchText : String = ""
-
+    @State var selected = 0
 
     var body: some View {
-        NavigationView {
-            VStack {
-                SearchBar(text: $searchText)
-                List {
-                    ForEach(hymns.filter { self.searchText.isEmpty ? true : $0.contains(self.searchText)}, id: \.self) { hymn in
-                                        Text(hymn)
-                    }
-                }.navigationBarTitle(Text("Look up any hymn"))
-            }
+        TabView(selection: $selected) {
+            //systemName: basically means they are coming from SF Icons.
+            SearchView().tabItem {
+                Image(systemName: "magnifyingglass")
```

Comment:
I think these SF icons look fine. If these are the recommended iOS icons, then we should just use that to maintain a consistent feel across the platform. No need to import custom icons if there are provided ones :)

Reply:
Sweet! I agree!